### ISSUE TITLE
Update for BCC

### ISSUE THREAD

**Issue by Kristog-D** [tullamods/Bagnon#1368]
Is there a new update for BCC??? as my classic Bagnon is working fine but on BCC its saying Out of Date in my addons and doesn't work... 

as current game is version 2.5.3 and Bagnon is 2.5.2

Many Thanks

**Comment by jazminite** [comment]
Seconded, BCC is live and it's broken:
![image](https://user-images.githubusercontent.com/65823478/159568688-69eba508-1b03-4a46-bffd-4780cba0b09c.png)

**Comment by TheShrubber** [comment]
Yeah it's unusable now, I get so many LUA errors trying to find retail items that I can even open the addon control panel to disable the addon.  Had to just CTRL-ALT-Delete to get out.  I noticed this issue with the last patch but I guess it got a band-aid fix?

**Comment by jazminite** [comment]
Looks like #1369 is addressing this
